### PR TITLE
Add script to execute sqllogictest files

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by mfussenegger** [comment]
@matriv please review

File: blackbox/sqllogictest/src/sqllogictest.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+        The result itself is either:
+
+         - The rows transformed to have a single column
+
+            Example:
+
+                2 rows with 2 columns:
+
+                    a| b
```
Comment (matriv) [inline blackbox/sqllogictest/src/sqllogictest.py]: fix indentation to match the Becomes part (line 109)

File: blackbox/sqllogictest/src/sqllogictest.py [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    if values != expected_values:
+        raise IncorrectResult(
+            'Expected {0} values, got {1}'.format(expected_values, values))
+    m = md5()
+    for row in rows:
+        m.update('{0}'.format(row).encode('ascii'))
+        m.update('\n'.encode('ascii'))
+    digest = m.hexdigest()
+    if digest != hash_:
+        raise IncorrectResult('Expected values hashing to {0}. Got {1}\n{2}'.format(
```
Comment (matriv) [inline blackbox/sqllogictest/src/sqllogictest.py]: Is it useful to output the hash results?

File: blackbox/sqllogictest/src/sqllogictest.py (hunk 1 above)
Comment (mfussenegger) [inline blackbox/sqllogictest/src/sqllogictest.py]: it already matches?

File: blackbox/sqllogictest/src/sqllogictest.py (hunk 2 above)
Comment (mfussenegger) [inline blackbox/sqllogictest/src/sqllogictest.py]: Not sure but I don't think it hurts

**Comment by mfussenegger** [comment]
merged to master

**Pull request closed without merging**